src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  Box,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormHelperText,
+  IconButton,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  TableCell,
+  TableRow,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import CancelIcon from '@mui/icons-material/Cancel';
+import { DatePicker } from '@mui/x-date-pickers';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import prettyBytes from 'pretty-bytes';
+import { Field, FieldProps, ErrorMessage, useFormikContext } from 'formik';
+import { AssociatedFile, LicenseNames, licenses } from '../../../types/associatedArtifact.types';
+import { ConfirmDialog } from '../../../components/ConfirmDialog';
+import { SpecificFileFieldNames } from '../../../types/publicationFieldNames';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { TruncatableTypography } from '../../../components/TruncatableTypography';
+
+interface FilesTableRowProps {
+  file: AssociatedFile;
+  removeFile: () => void;
+  toggleLicenseModal: () => void;
+  baseFieldName: string;
+}
+
+export const FilesTableRow = ({ file, removeFile, baseFieldName }: FilesTableRowProps) => {
+  const { t } = useTranslation();
+  const [openConfirmDialog, setOpenConfirmDialog] = useState(false);
+  const toggleOpenConfirmDialog = () => setOpenConfirmDialog(!openConfirmDialog);
+  const { setFieldValue, setFieldTouched } = useFormikContext();
+
+  return (
+    <TableRow data-testid={dataTestId.registrationWizard.files.fileRow}>
+      <TableCell sx={{ wordBreak: 'break-word', minWidth: '8rem' }}>
+        <TruncatableTypography>{file.name}</TruncatableTypography>
+      </TableCell>
```

Comment:
Usikker på denne, men lurer på om TruncatedTypography håndterer wordBreak selv, slik at denne ytterste her ikke har noen effekt?
```suggestion
      <TableCell sx={{ minWidth: '8rem' }}>
        <TruncatableTypography>{file.name}</TruncatableTypography>
      </TableCell>
```

Reply:
Truncated håndterer kun setninger, virker det som. 
![image](https://user-images.githubusercontent.com/53341294/204000094-da94bc52-9551-4bba-b529-a8a88b120d6a.png)